tests/conftest.py

```diff
@@ -208,3 +209,13 @@ def bundle_archive(request, tmp_path):
     Prepare tar file from dir without using bundle packer
     """
     return _pack_bundle(request.param, tmp_path)
+
+
+@pytest.fixture(scope="function")
+def auth_to_adcm(app_fs, adcm_credentials):
+    """Perform login on Login page ADCM
+    :param app_fs:
+    :param adcm_credentials:
+    """
```

Comment:
Remove default scope.
Add type hints and remove `:param` directives.

Reply:
removed